src/data/events.json

```diff
@@ -0,0 +1 @@
 to use git and participate in a speed friending to find a team for Hackillinois 2023!","startTime":1677294000,"endTime":1677299400,"locations":[{"description":"Siebel 2405","tags":[],"latitude":40.113812,"longitude":-88.224937}],"sponsor":"","eventType":"WORKSHOP","points":100,"isAsync":false,"isPrivate":false,"displayOnStaffCheckIn":false},{"id":"f9cff398e5752bf40a23ba59f67013ae","name":"Project Submission Deadline","description":"Projects are due! Make sure to get them submitted on Devpost.","startTime":1677423600,"endTime":1677423600,"locations":[],"sponsor":"","eventType":"OTHER","points":0,"isAsync":false,"isPrivate":false},{"id":"4f3728473ddcf8dc6024e082194830be","name":"Scavenger Hunt 7","description":"All that detective work and stair-ascending have tired you out, and all you want is to be able to sit down somewhere comfortable. Not anywhere too far from where you are right now though, because staircases are too much work right now.","startTime":1677272400,"endTime":1677281400,"locations":[],"sponsor":"","eventType":"OTHER","points":10,"isAsync":false,"isPrivate":true,"displayOnStaffCheckIn":false},{"id":"8bcf8f86bdcba9e035893268aa405bbf","name":"John Deere Scavenger Hunt Stop ","description":"Expo Scavenger Hunt Stop ","startTime":1677276000,"endTime":1677281400,"locations":[{"description":"Siebel Scavenger Hunt Stop ","tags":["SIEBEL1"],"latitude":40.113812,"longitude":-88.224937}],"sponsor":"John Deere","eventType":"OTHER","points":25,"isAsync":false,"isPrivate":true,"displayOnStaffCheckIn":false},{"id":"df6e1692b5f70980e1da70833b1eb7f2","name":"Epic Scavenger Hunt Stop ","description":"Expo Scavenger Hunt Stop ","startTime":1677276000,"endTime":1677281400,"locations":[{"description":"Siebel ","tags":["SIEBEL1"],"latitude":40.113812,"longitude":-88.224937}],"sponsor":"Epic","eventType":"OTHER","points":25,"isAsync":false,"isPrivate":true,"displayOnStaffCheckIn":false},{"id":"fdffd8ca23b8241d263303f1d1c98bfd","name":"John Deere Lounge","description":"Come to CIF 3038 to hang out with John Deere and chat about opportunities and anything else from 8:00 AM to 10:00 PM.","startTime":1677333600,"endTime":1677384000,"locations":[{"description":"CIF 3038","tags":[],"latitude":40.11264,"longitude":-88.22832}],"sponsor":"John Deere","eventType":"OTHER","points":0,"isAsync":false,"isPrivate":false,"displayOnStaffCheckIn":false},{"id":"9699f6902174530bc775abc7918bd29f","name":"Scavenger Hunt 8","description":"You venture outdoors and see a burst of twinkling lights and fanfare in the far distance (0.2 miles southeast from your current location, to be exact). Could it be that this carnival is split up into two separate locations? You decide to check it out and somehow wind up on the 3rd floor, in the place where you may see yourself 10 years from now.\n\nHINT: think about clue 4!","startTime":1677272400,"endTime":1677281400,"locations":[],"sponsor":"","eventType":"OTHER","points":10,"isAsync":false,"isPrivate":true,"displayOnStaffCheckIn":false}]}
```

Comment:
I suggest deleting the events marked `isPrivate`. Even though our stuff is open-source and people can see that events can be private, I lean towards hiding it since the public API endpoint never returned private events.

Reply:
Yeah I was considering that but then thought that since these events are all scavenger hunt related, they don't actually need to be private anymore after the scavenger hunt ended. I'm a data hoarder 😅 and think it might be nice to have all the events be easily accessible in the repo (while still only displaying the public ones for parity).

But completely up to you, definitely makes sense to remove the private events as well since originally the frontend would have never even seen them.

Reply:
Both options are reasonable but I'm leaning more towards deleting them because those events were marked as private for a reason, even if the time is passed we should preserve that design. Also the intent is to archive the site as it appeared during the event, and it looks like past websites/repos don't have private events. 

Looks great overall though, thank you Abhi! We can get this merged after removing those.

Reply:
sounds good!